### PR TITLE
SSH/SCP for CI environment

### PR DESCRIPTION
**Pull request by morses-code**
- added fix to allow ssm/ic for ci
- updated example_config.yml

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -151,14 +154,21 @@ func (cfg Config) GetMyIP() (string, error) {
...
-	for _, v := range s {
-		if v == str {
-			return true
-		}
+func (cfg Config) GetPath(env Environment) string {
+	if env.CI {
+		return cfg.DPCIPath
+	} else {
+		return cfg.DPSetupPath
+	}
```
Comment (daiLlew) [inline config/config.go:165]: Else not required as its implied if `if env.CI` is not true.

File: config/config.go [hunk 2]
```diff
@@ -151,14 +154,21 @@ func (cfg Config) GetMyIP() (string, error) {
...
+}
+
+func (cfg Config) GetAnsibleDirectory(env Environment) string {
+	if env.CI {
+		return filepath.Join(cfg.DPCIPath, "ansible")
+	} else {
+		return filepath.Join(cfg.DPSetupPath, "ansible")
```
Comment (daiLlew) [inline config/config.go:172]: ditto

File: config/config.go [hunk 3]
```diff
@@ -15,7 +15,7 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
-var awsbEnvs = []string{"dp-sandbox", "dp-prod", "dp-ci"}
+const AWSB_TAG = "awsb"
```
Comment (daiLlew) [inline config/config.go:18]: Godoc.

File: command/scp.go [hunk 4]
```diff
@@ -72,7 +72,9 @@ func createEnvironmentSCPSubCommands(cfg *config.Config, scpOpts scp.Options) ([
 
 // create an array of environment group sub-commands available to `scp env`
 func createEnvironmentGroupSCPSubCommands(env config.Environment, cfg *config.Config, scpOpts scp.Options) ([]*cobra.Command, error) {
-	groups, err := ansible.GetGroupsForEnvironment(cfg.DPSetupPath, env.Name)
+	path := cfg.GetPath(env)
+
+	groups, err := ansible.GetGroupsForEnvironment(path, env.Name)
```
Comment (daiLlew) [inline command/scp.go:77]: Will this handle empty path?

Comment (morses-code) [reply command/scp.go:77]: As it defaults to DPSetupPath in the config its fine. When this isn't set in dp-cli-config.yml the application will warn and not create the group commands.

**Review by daiLlew — APPROVED** [review]

**Pull request merged**